### PR TITLE
added crypto-dashboard

### PR DESCRIPTION
**Pull request by Aditijainnn**
Closes: #785

- **Title:** Crypto Dasboard
- **Name:** Aditi
- **Idenitfy yourself:**  Gssoc ext and Hacktoberfest 2024

### Screenshots 📷

372829766-e21ec12f-da3b-4917-8a6e-0a9a71d378c8

Example how to mark a checkbox:-
- [x] My code follows the code style of this project.
-->
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### Checklist: ☑️

- [x] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CODE_OF_CONDUCT.md) of this project.
- [x] This PR does not contain plagiarized content.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] My changes generate no new warnings.

### CODE REVIEW

File: Vue-JS-Projects/Intermediate/Crypto-Dashboard/README.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+```
+al
+
+
+<!-- -------------------------------------------------------------------------------------------------------------- -->
+
+## :zap: Screenshots 📸
+<!-- add the screenshot of the project (Mandatory) -->
+
+<img src='./screenshot.webp.png'>
```
Comment (Avdhesh-Varshney) [inline Vue-JS-Projects/Intermediate/Crypto-Dashboard/README.md:74]: There is a mis-understanding. You have to convert your png file into webp format. That's why guidelines says `screenshot.webp` not `screenshot.png` and not `screenshot.webp.png`.

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]

**Comment by Avdhesh-Varshney** [comment]
https://github.com/JayeshLab/vue-crypto-dashboard

**Comment by Aditijainnn** [comment]
@Avdhesh-Varshney please can you remove the spam tag because due to this got disqualied from hacktoberfest 
please i will not repeat this again can you remove spam tag

**Comment by Aditijainnn** [comment]
> Don't repeat it next time

OK thank you sir

**Pull request closed without merging**